_posts/2018-05-14-coding-period-begins.md

```diff
@@ -0,0 +1,42 @@
+---
+    date: 2018-05-14
+    category: Bonding Period
+---
+
+##### Bonding Period Ends
+26 April 2018 - 13 May 2018
+
+<br/><br/>
+Now that the bonding period has finally come to an end, I thought
+I should let you know about the developments during the period and
+my plan ahead. Bonding Period is the time meant to help the students
+get started with the community, coding practices and discuss the details
+of the project one plans to follow with the community and their mentors.
+I had a great time knowing the community and the interesting stuff they do.
+
+<br/><br/>
+> ##### Developments
+
+1. **Resolved pending PR's**: 
+  I joined the coala community in late March and made a few 
+  contributions to projects.coala.io and coala-vs-code of 
+  which some were left pending in review. I got all of them 
+  resolved and merged.<br/><br/>
+
+2. **Finish newcomer process**: 
+  Since the deadline was too close when I joined the community 
+  I was instrcuted to work on my proposal first and worry with 
+  the newcomer process later on. I have since completed the 
+  newcomer process.<br/><br/>
+
+3. **Schedule**: 
+  I worked with my primary(@gaocegege) and we have decided on a week 
+  on week schedule that I should follow to stay on track.<br/><br/>
+
+4. **Work flow**: 
+  We have discussed about the work flow I need to maintain and things
+  such as tooling etc.
+
```

Comment:
And we also update the details in projects.coala.io. It also takes us some time. I think we could add it here

Reply:
> It also takes us some time.

What do you mean?

Reply:
I mean we need to list all you have done in the community bonding process. Then we need to list the update in projects.coala.io.

I am not sure if I express correctly. Do you get me? :thinking:

Reply:
Yes, you mean that we need to post a reference to this list in projects.coala.io?

Reply:
I have also updated this with a reference to projects.coala.io.

Reply:
Yeah. For example.

```markdown
5. Project description update

...
```